my-sms/app/views/students/index.html.erb

```diff
@@ -1,35 +1,46 @@
-<h1>Listing students</h1>
-
-<table>
-  <tr>
-    <th>Title</th>
-    <th>First name</th>
-    <th>Middle name</th>
-    <th>Last name</th>
-    <th>Email</th>
-    <th>Birth date</th>
-    <th>Gender</th>
-    <th></th>
-    <th></th>
-    <th></th>
-  </tr>
-
-<% @students.each do |student| %>
-  <tr>
-    <td><%= student.title %></td>
-    <td><%= student.first_name %></td>
-    <td><%= student.middle_name %></td>
-    <td><%= student.last_name %></td>
-    <td><%= student.email %></td>
-    <td><%= student.birth_date %></td>
-    <td><%= student.gender %></td>
-    <td><%= link_to 'Show', student %></td>
-    <td><%= link_to 'Edit', edit_student_path(student) %></td>
-    <td><%= link_to 'Destroy', student, confirm: 'Are you sure?', method: :delete %></td>
-  </tr>
-<% end %>
+<%- model_class = Student -%>
+<div class="page-header">
+  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+</div>
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th><%= model_class.human_attribute_name(:id) %></th>
+      <th><%= model_class.human_attribute_name('name') %></th>
+      <th><%= model_class.human_attribute_name(:email) %></th>
+      <th><%= model_class.human_attribute_name(:birth_date) %></th>
+      <th><%= model_class.human_attribute_name(:gender) %></th>
+      <th><%= model_class.human_attribute_name(:created_at) %></th>
+      <th><%=t '.actions', :default => t("helpers.actions") %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @students.each do |student| %>
+      <tr>
+        <td><%= link_to student.id, student_path(student) %></td>
+        <td><%= student.full_name %></td>
+        <td><%= student.email %></td>
+        <td><%= student.birth_date %></td>
+        <td><%= student.gender %></td>
+        <td><%=l student.created_at %></td>
+        <td>
+          <%= link_to 'Show', student, class: 'btn btn-default btn-xs' %>
+          <%= link_to t('.edit', :default => t("helpers.links.edit")),
+                      edit_student_path(student), :class => 'btn btn-default btn-xs' %>
+          <%= link_to t('.destroy', :default => t("helpers.links.destroy")),
+                      student_path(student),
+                      :method => :delete,
+                      :data => { :confirm => t('.confirm', :default => t("helpers.links.confirm", :default => 'Are you sure?')) },
+                      :class => 'btn btn-xs btn-danger' %>
```

Comment:
Minor - but in the plus repo we prefer to use the `{ key: value }` syntax over the `{ :key => value }` syntax for hashes

Reply:
Fixed in 5554e5f766237ceea0344d79b67ba1becda28022